### ISSUE TITLE
Separate Loss function result differs from training Loss result

### ISSUE THREAD

**Issue by jklanger** [DeepMLNet/DeepNet#30]
Loss evaluated with a separate function:
`let lossFn = mi.Func<single> (loss) |> arg2 input target`

differs from the loss evaluated during training:
`let trainFn () = Train.train trainable fullDataset trainCfg`

When thisoption ist set:
`SymTensor.Debug.DisableCombineIntoElementsOptimization <- true`
the results are consistent again!

Tested via DeepPrivate:
`Z:\DeepPrivate\****Models\bin\Release\****Models.exe Z:\DEVELOP\DeepPrivate\****Models\cfgs\COPYING\LSTM\80reluOutputTest\Config.fsx`

Code during training gets optimized differently?
